catalog/core/catalog-core-migratable/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -39,6 +39,6 @@
     </bean>
 
```

Comment:
It doesn't look like you are injecting the new parameters for the constructor in your blueprint.